### PR TITLE
Updated README and templates

### PR DESCRIPTION
**Pull request by MinnDevelopment**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

There are several guidelines you should follow in order for your
  Pull Request to be merged.

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

> It is sometimes better to include more changes in a single commit. 
  If you find yourself having an overwhelming amount of commits, you
  can **rebase** your branch.

## Description

We are changing our branch model. These changes reflect the new guidelines and cleans up some of the documentation and templates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,6 +186,49 @@ repositories {
...
+```xml
+<dependency>
+    <groupId>com.github.DV8FromTheWorld</groupId>
+    <artifactId>JDA</artifactId>
+    <version>COMMIT</version>
+</dependency>
+```
+```xml
+<repository>
+    <id>jcenter</id>
```
Comment (Almighty-Alpaca) [inline README.md]: These builds will only be on jitpack, not on jcenter.

File: README.md [hunk 2]
```diff
@@ -184,6 +186,49 @@ repositories {
...
+**Gradle**
+```gradle
+dependencies {
+    compile 'com.github.DV8FromTheWorld:JDA:COMMIT'
+}
+
+repositories {
+    jcenter()
+    maven {
+        name 'jitpack'
```
Comment (Almighty-Alpaca) [inline README.md]: Same as above.

File: README.md (hunk 1 above)
Comment (MinnDevelopment) [reply README.md]: jcenter is added for transitive dependencies which are not resolvable from jitpack

Comment (Almighty-Alpaca) [reply README.md]: right...

**Comment by MinnDevelopment** [comment]
Should we bump version in this PR?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -14,13 +14,15 @@ Hey there! Before you report a bug or suggest a new feature,
 > This is not the place to learn Java. Please refer to [StackOverflow][stack overflow]
   for your general programming questions.
 
-## Template
+## Description
```
Comment (kantenkugel) [inline .github/ISSUE_TEMPLATE.md]: Maybe use "Issue" instead to avoid using "Description" twice?

File: README.md [hunk 4]
```diff
@@ -151,27 +151,30 @@ In addition you can look at the many Discord Bots that were implemented using JD
 [And many more!](https://github.com/search?q=JDA+discord+bot&type=Repositories&utf8=%E2%9C%93)
 
 ## Download
+Latest Stable Version: [GitHub Release](https://github.com/DV8FromTheWorld/JDA/releases/latest)
```
Comment (kantenkugel) [inline README.md]: not a fan of advertizing releases as builds to use

File: src/examples/java/MessageListenerExample.java [hunk 5]
```diff
@@ -214,15 +214,18 @@ else if (msg.startsWith("!kick"))   //Note, I used "startsWith, not equals.
                         // in the Role hierarchy than they are. Remember, NO ONE is above the Guild's Owner. (Guild#getOwner())
                         if (!selfMember.canInteract(member))
                         {
-                            channel.sendMessage("Cannot kicked member: " + member.getEffectiveName() +", they are higher " +
-                                    "in the hierachy than I am!").queue();
+                            // use the MessageAction to construct the content in StringBuilder syntax using append calls
+                            channel.sendMessage("Cannot kicked member: ")
```
Comment (kantenkugel) [inline src/examples/java/MessageListenerExample.java]: Lol nice grammar... I know it was there previously, but might still fix

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (MinnDevelopment) [reply .github/ISSUE_TEMPLATE.md]: Maybe thats better yeah

File: README.md (hunk 4 above)
Comment (MinnDevelopment) [reply README.md]: The point is to have a stable release build, dev builds can have a variance of changes that are difficult to find. Stable releases allow to see a full changelist and adapt more easily

**Pull request merged**